app/assets/javascripts/Components/Modals/create_modify_annotation_panel_modal.jsx

```diff
@@ -29,17 +29,23 @@ class CreateModifyAnnotationPanel extends React.Component {
       return;
     }
 
-    $.ajax({
-      method: "get",
-      url: Routes.find_annotation_text_course_assignment_annotation_categories_url(
-        this.props.course_id,
-        this.props.assignment_id
-      ),
-      dataType: "json",
-      data: {
-        string: value,
+    const url = Routes.find_annotation_text_course_assignment_annotation_categories_url(
+      this.props.course_id,
+      this.props.assignment_id
+    );
+    const requestUrl = `${url}?string=${value}`;
+
+    fetch(requestUrl, {
+      method: "GET",
+      headers: {
+        Accept: "application/json",
       },
     })
+      .then(response => {
+        if (response.ok) {
+          return response.json(); // Parse the response as JSON
```

Comment:
Delete the comment here (and elsewhere you included this or similar comments); it is clear from the code